src/js/playlist/playlist.js

```diff
@@ -3,10 +3,12 @@ import PlaylistItem from 'playlist/item';
 import Source from 'playlist/source';
 import Providers from 'providers/providers';
 import { PlayerError, MSG_CANT_PLAY_VIDEO } from 'api/errors';
+import { isObject } from 'utils/underscore';
 
-const Playlist = function(playlist) {
+const Playlist = function(playlist, model) {
```

Comment:
I passed the model to the Playlist because the 'liveSyncDuration' attribute is set in the model, but it needs to be copied to the playlist items.

Reply:
It should be passed from the config, not the model.